opentreemap/treemap/js/src/CanopyFilterControl.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+var $ = require('jquery'),
+    _ = require('lodash'),
+    L = require('leaflet'),
+    Bacon = require('baconjs');
+
+require('ionRangeSlider');
+
+var canopyFilterTmpl = _.template($('#canopy-filter-tmpl').html());
+
+var CanopyFilterControl = L.Control.extend({
+    options: {
+        position: 'bottomleft'
+    },
+
+    initialize: function() {
+        this._changeBus = new Bacon.Bus();
+        this.tilerArgsProp = this._changeBus.toProperty({
+            category: 'Municipality',
```

Comment:
My thoughts are that we could reasonably extend the canopy filter to allow users to select which category to display. In that case, I think it's best to leave the code as-is. On the other hand, if we will only ever have the option to display a single category per instance than your solution makes more sense. 

Another alternative is to dynamically adjust which features are displayed based on zoom level. In which case, neither of these solutions matter. I decided that it might make more sense to just have some hard-coded assumptions for now, and wait to see how the requirements for this feature evolve.

If you have a strong preference for pulling this value out into the instance settings, I can do that. But it will still be hard-coded, just at a slightly higher level.